index.html

```diff
@@ -8,12 +8,110 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <title> Judith Ricketts | Karma Code  | </title> <!-- the document's title that is shown in a browser's title bar or a page's tab -->
+</head>
+
+<body>
+
+  <header>
+    <nav class="top-navigation">
+      <div class="content1">
```

Comment:
Is this `div` necessary? You could try removing it and adding the class name on the `nav` element above... it's good to remove any `div`s you can so your HTML is as semantic as possible.